less/gmcq.less

```diff
@@ -179,6 +179,11 @@
         }
     }
 
+    .gmcq-item-graphic-attribution {
+        font-size:0.75em;
+        line-height: 1em;
```

Comment:
@tomgreenfield This style is an exact copy of the attribution code already in adapt-contrib-graphic/less/graphic.less. Happy to change, but I thought i'd keep it consistent to something already in the code base.